src/util/container.h

```diff
@@ -492,6 +501,7 @@ class ModifySafeMap
 	std::map<K, V> m_new;
 	unsigned int m_iterating = 0;
 	size_t m_garbage = 0; // upper bound for null-placeholders in m_values
+	const V m_null = V();
```

Comment:
Should probably be `static const`.

Reply:
I would have liked that but it throws lots of c++ errors

Reply:
Ah, right. Stupid C++ rules.
You have to use `inline static const`.
<!--
`inline static const` works though, but only if `V()` is constexpr (which is luckily true for `unique_ptr`s).
Otherwise one could also use a local static variable:
```cpp
static const V &getMNull() { static const V null = V(); return null; }
```
-->
But we don't use this datastructure often anyway, so a non-static member would also be fine. *shrug*